src/schema/v2/markdownContent.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { markdown } from "schema/v2/fields/markdown"
+
+const MarkdownContentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "MarkdownContent",
+  fields: {
+    content: markdown(({ content }) => content),
+  },
+})
+
+export const MarkdownContent: GraphQLFieldConfig<void, ResolverContext> = {
+  type: new GraphQLNonNull(MarkdownContentType),
+  args: {
+    content: { type: new GraphQLNonNull(GraphQLString) },
+  },
+  resolve: async (_source, { content }) => {
+    return { content }
```

Comment:
For security purposes, I think it might be nice to hit `meLoader()`, verify it's success (user's token is valid), _and_ then only allow `content_manager` roles to do this. Otherwise it's kind of like anyone can pass huge strings in for translation.